website/pages/docs/agent/options.mdx

```diff
@@ -814,8 +814,16 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
 
 - `advertise_addr` Equivalent to the [`-advertise` command-line flag](#_advertise).
 
```

Comment:
Were these new options HCL only (as in not available through CLI options?)

Reply:
yes, config file only (json and hcl).